### DISCUSSION TITLE
mmex 1.5.21 release time

### DISCUSSION THREAD

**Discussion by vomikan** [moneymanagerex/moneymanagerex#5081] [General]
@whalley @n-stein it seems time to release 1.5.21
I've updated translations. Please resolve all small cases if possible.
More serious changes are better left for release 1.5.22

Regards,
Nikolay

**Comment by n-stein** [comment]
#4969 can be marked as fixed

**Reply by whalley** [reply]
> #4969 can be marked as fixed

Done.

**Comment by whalley** [comment]
Think I'm up-to-date with all the changes I was planning.

**Comment by vomikan** [comment]
Translations are expected in most languages.
![image](https://user-images.githubusercontent.com/6836805/190604834-9af32989-1a49-4fe4-a8cc-86ef91c6a693.png)
